spec/rubocop/server/cache_spec.rb

```diff
@@ -241,4 +241,17 @@
       end
     end
   end
+
+  if RuboCop::Server.support_server?
+    describe '.pid_running?' do
```

Comment:
It looks like `:isolated_environment` is needed because `described_class.dir.rmtree` manipulates the real cache directory.

```suggestion
    describe '.pid_running?', :isolated_environment do
```